freshdesk/manifest.json

```diff
@@ -1,25 +1,41 @@
 {
-  "platform-version": "2.0",
-  "product": {
-    "freshdesk": {
+  "platform-version": "3.0",
+  "modules": {
+    "common": {
       "location": {
         "full_page_app": {
           "url": "views/fullpage.html",
           "icon": "styles/images/icon.svg"
-        },
+        }
+      }
+    },
+    "support_ticket": {
```

Comment:
Can we add other products' data methods sample app also in a single app for 3.0?